Web/CSS2/styleLeaf.css

```diff
@@ -0,0 +1,18 @@
+
+.hemisphere {
+width:200px;
+height:120px;
+background-color:#2EB82E;
+border-radius:0 120px 0 120px;
```

Comment:
Border-radius can have one, two, three or four arguments, in your case it can be achieved by giving it two arguments.